LayoutTests/TestExpectations

```diff
@@ -7602,6 +7602,9 @@ imported/w3c/web-platform-tests/css/css-anchor-position/position-visibility-no-o
 imported/w3c/web-platform-tests/css/css-anchor-position/position-visibility-remove-anchors-visible.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-anchor-position/position-visibility-remove-no-overflow.html [ ImageOnlyFailure ]
 
+# anchor HTML attribute
+imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/anchor.html [ Failure ]
+
```

Comment:
This line shouldn't be necessary I think, because the `anchor-expected.txt` should be sufficient here.

Reply:
It's not strictly necessary now that I've added the platform-specific expectations, but the extra failures on those platforms made me want to add it - the failures are specific about positions which are only the way they are because the attribute is not yet implemented. Ideally we shouldn't need platform-specific expectations for this test at all, I think.